packages/react-native-gesture-handler/package.json

```diff
@@ -8,11 +8,12 @@
     "build": "yarn tsc -p tsconfig.build.json",
     "precommit": "lint-staged",
     "ts-check": "yarn tsc --noEmit",
-    "format:js": "prettier --write --list-different './{src,example,FabricExample,MacOSExample}/**/*.{js,jsx,ts,tsx}'",
+    "format-js": "prettier --write --list-different './src/**/*.{js,jsx,ts,tsx}'",
+    "format:js": "prettier --write --list-different './src/**/*.{js,jsx,ts,tsx}'",
```

Comment:
Can we make this (and `lint:js`) call the other command with `-`?

Reply:
Sure, done in [18ce1a6](https://github.com/software-mansion/react-native-gesture-handler/pull/3503/commits/18ce1a69497a6d96bbd26cb9ec1f179ed3fff189).